server/schema.graphql

```diff
@@ -339,12 +351,16 @@ interface DeleteRequestRequisitionLineErrorInterface {
 input DeleteRequestRequisitionLineInput {
   id: String!
 }
-union DeleteRequestRequisitionLineResponse = DeleteRequestRequisitionLineError | DeleteResponse
+union DeleteRequestRequisitionLineResponse =
+    DeleteRequestRequisitionLineError
+  | DeleteResponse
 type DeleteRequestRequisitionLineResponseWithId {
   id: String!
   response: DeleteRequestRequisitionLineResponse!
 }
-union DeleteRequestRequisitionResponse = DeleteRequestRequisitionError | DeleteResponse
+union DeleteRequestRequisitionResponse =
```

Comment:
are you using prettier? or some other formatter? it makes the review harder than it needs to be!

Reply:
Prettier. Sorry, didn't realise it formatted here

Reply:
of course it's possible that we both have the same prettier config and what is in the PR is the 'correct' formatting 😁 
All good - my concern is that one of us has a different config and we end up in a format war every time a file is touched.. let's just keep an eye on it.